nixos/modules/programs/regreet.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, pkgs
+, config
+, ...
+}:
+let
+  cfg = config.programs.regreet;
+  settingsFormat = pkgs.formats.toml { };
+in
+{
+  options.programs.regreet = {
+    enable = lib.mkEnableOption (lib.mdDoc "") // {
```

Comment:
What is the `//` for? If this is a mistake: value of `description` is just a string, not a markdown doc-string. The first parameter of `lib.mkEnableOption` is `lib.mdDoc ""` which is overwritten by `description`'s regular string.

If you wanted to `//` with an attrset so that `description` would be there for symmetry sake, `description = lib.mdDoc ''...` and `lib.mkEnableOption null // {...` would work.

But, I think this is a mistake because `package = lib.mkPackageOptionMD` is also asymmetrical with the rest of the `lib.mkOption`s.

Reply:
I clearly needed sleep. I forgot to add `lib.mdDoc` before `''`. Couldn't figure out what was wrong. Thanks.